src/main/resources/application.properties

```diff
@@ -1,12 +1,3 @@
-server.port=8090
-spring.jpa.database=POSTGRESQL
-spring.datasource.platform=postgres
-spring.datasource.url=jdbc:postgresql://localhost:5432/student-info
-spring.datasource.username=postgres
-spring.datasource.password=password
-spring.jpa.show-sql=true
-spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto=update
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
-spring.jpa.properties.hibernate.format_sql=true
+spring.datasource.username=
+spring.datasource.password=
+
```

Comment:
Since you added *.properties, we no longer need this. Delete this file completely while committing. 

Just keep this copy somewhere in local with proper username and password. After the PR merge you can add the file back in local and git will no longer track it.